### PR TITLE
Adding Bottomsheet guide to the barometer

### LINKED ISSUE #1245: Adding bottomsheet for the Barometer
**Actual Behaviour**
Right now there is no bottomsheet for the Barometer instrument

**Expected Behaviour**
Bottomsheet guide and the schematic must be present for the barometer

**Steps to reproduce it**
Open Barometer and the slide up the bottomsheet

**LogCat for the issue**
N/A

**Screenshots of the issue**
N/A

**Would you like to work on the issue?**
Yes, but I would require @CloudyPadmal help for the schematic of the connections due to lack of device.

### PR DESCRIPTION
**Pull request by yatri1609**
Fixes #1245 

**Changes**: 
Added bottomsheet to barometer experiment

**Screenshot/s for the changes**: 

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**:
[barometer_bottomsheet.zip](https://github.com/fossasia/pslab-android/files/2271443/barometer_bottomsheet.zip)

### CODE REVIEW

**Comment by yatri1609** [comment]
@CloudyPadmal @abhinavraj23 @Avjeet @harsh-2711 Please review it.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -895,8 +895,14 @@
...
 
-    <string name="barometer_description">Measures the pressure of the atmosphere</string>
     <string name="barometer">Barometer</string>
+    <string name="barometer_description">Measures the atmospheric pressure</string>
+    <string name="barometer_intro">\u2022 The Barometer can be used to measure the atmospheric pressure. This instruments
+    is compatible with either the built in pressure sensor on any android device or the BMP-180 pressure sensor.\n\n
+    \u2022 If you want to use the sensor BMP-180, connect the sensor to PSLab device as shown below</string>
+    <string name="barometer_desc">\u2022 The above pin configuration has to be same except for the pin
+    GND. GND is meant for Ground and any of the PSLab device GND pins can be used since they are common.\n\n
+    \u2022 Select sensor by going to the Configure tab from the bottom navigation bar and choose BMP-180 in the drop down menu under Select Sensor.</string>
```
Comment (CloudyPadmal) [inline app/src/main/res/values/strings.xml]: Select the sensor

Comment (CloudyPadmal) [inline app/src/main/res/values/strings.xml]: Can we emphazise "Configure" and "Select Sensor"?

**Comment by yatri1609** [comment]
@CloudyPadmal I'll make the requested changes

**Pull request merged**